pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -378,6 +378,7 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
+		klog.V(2).Infof("victim %v/%v preempted by %v/%v on node %v", victim.Namespace, victim.Name, pod.Namespace, pod.Name, c.Name())
```

Comment:
Please follow the guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Reply:
> Please follow the guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

I have read the guidelines and made the corrections